docs/visualize/regionmap.asciidoc

```diff
@@ -0,0 +1,38 @@
+[[regionmap]]
+== Region Maps
+
+Region maps are thematic maps in which boundary vector shapes are colored using a gradient, with higher intensity colors indicating larger values and lower intensity colors indicating smaller values. 
+These are also known as choropleth maps. 
+
```

Comment:
I'd tweak the first sentence slightly:

Region maps are thematic maps in which boundary vector shapes are colored using a gradient: higher intensity colors indicate larger values, and lower intensity colors indicate smaller values.